webservices/resources/sched_a.py

```diff
@@ -118,10 +119,16 @@ def build_query(self, **kwargs):
         if kwargs.get('sub_id'):
             query = query.filter_by(sub_id=int(kwargs.get('sub_id')))
         if kwargs.get('line_number'):
+            # line_number is a composite value of 'filing_form-line_number'
             if len(kwargs.get('line_number').split('-')) == 2:
                 form, line_no = kwargs.get('line_number').split('-')
                 query = query.filter_by(filing_form=form.upper())
                 query = query.filter_by(line_number=line_no)
+            else:
+                raise exceptions.ApiError(
+                    "Invalid line_number detected.",
```

Comment:
Can we tell the user what the required line number format is?

Reply:
updated. thx.